ui/app/AppLayouts/Chat/popups/community/BurnTokensPopup.qml

```diff
@@ -0,0 +1,177 @@
+import QtQuick 2.15
+import QtQuick.Controls 2.14
+import QtQuick.Layouts 1.14
+import QtQml.Models 2.14
+import QtGraphicalEffects 1.0
+
+import StatusQ.Core 0.1
+import StatusQ.Controls 0.1
+import StatusQ.Popups.Dialog 0.1
+import StatusQ.Core.Theme 0.1
+import StatusQ.Core.Utils 0.1
+import StatusQ.Controls.Validators 0.1
+import StatusQ.Components 0.1
+
+import AppLayouts.Chat.panels.communities 1.0
+
+import utils 1.0
+
+StatusDialog {
+    id: root
+
+    property string communityName
+    property string tokenName
+    property int remainingTokens
+    property url tokenSource
+    property bool isAsset // If asset isAsset = true; if collectible --> isAsset = false
+
+    signal burnClicked(int burnAmount)
+    signal cancelClicked
+
+    QtObject {
+        id: d
+
+        property alias amountToBurn: amountToBurnInput.text
+
+        function initialize() {
+            specificAmountButton.checked = true
+            amountToBurnInput.forceActiveFocus()
+        }
+
+        function getVerticalPadding() {
+            return root.topPadding + root.bottomPadding
+        }
+
+        function getHorizontalPadding() {
+            return root.leftPadding + root.rightPadding
+        }
+    }
+
+    implicitWidth: 600 // by design
+    implicitHeight: content.implicitHeight + footer.height + header.height + d.getVerticalPadding()
```

Comment:
Not needed

Reply:
I want to make the heigh dynamic depending on the `contentItem` heigh, but also having into account header/footer, not a fixed value.